lib/ts/mock_station.ex

```diff
@@ -0,0 +1,21 @@
+defmodule MockStation do
```

Comment:
This file is being used in tests. Hence the right location for it is test/ts/mock_station.ex